### PR TITLE
allow for use of loginStandard functionality with docker login

### PR DESCRIPTION
**Pull request by dbeal-wiser**
Our pipelines actually supply the exact username and password without AWS credentials. I recognize this is unusual, but it should be possible to override the automatic detection functionality. This PR adds an option, `isECR`, which allows for the `loginStandard` functionality to be used even if ECR is detected.

Original behavior should not be altered unless user specifies `isECR` option.

suggest `Squash and Merge` for this PR.

### CODE REVIEW

**Comment by dbeal-wiser** [comment]
are the test failures because my PR is running from another repo?

**Review by mahdiamer — APPROVED** [review]

**Review by mahdiamer — APPROVED** [review]
~~~~

**Review by l2ichshop2 — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ inputs:
   registry:
     description: 'Server address of Docker registry. If not set then will default to Docker Hub'
     required: false
+  isECR:
```
Comment (crazy-max) [inline action.yml:13]: I think naming it `ecr` is enough

**Review by crazy-max — CHANGES_REQUESTED** [review]
Logs for failing tests are not available anymore. Can you rebase to trigger the CI again? Thx.

**Comment by crazy-max** [comment]
#123

**Pull request closed without merging**